Objects/longobject.c

```diff
@@ -5795,37 +5786,43 @@ _PyLong_Init(void)
         Py_SIZE(v) = size;
         v->ob_digit[0] = (digit)abs(ival);
 
-        small_ints[i] = v;
+        tstate->interp->small_ints[i] = v;
     }
 #endif
-    _PyLong_Zero = PyLong_FromLong(0);
-    if (_PyLong_Zero == NULL) {
-        return 0;
-    }
 
-    _PyLong_One = PyLong_FromLong(1);
-    if (_PyLong_One == NULL) {
-        return 0;
-    }
+    if (_Py_IsMainInterpreter(tstate)) {
```

Comment:
Yuck!

It would be great to have a comment here about how we would like to get rid of this special case.  Bonus points if you open an issue for that and link to it in the comment. :)

Reply:
I have no plan to remove _PyLong_One and _PyLong_Zero yet. It seems like there are too many things to do for subinterpreters, **I** am not excited by long TODO lists. They depress me, as if I will never be able to go to the end. I like to make tiny incremental changes :-)

In general, searching for "_Py_IsMainInterpreter" became a nice hint for "subinterpreters TODO".